demos/python_demos/text_spotting_demo/text_spotting_demo.py

```diff
@@ -307,7 +307,8 @@ def main():
 
         texts = []
         for feature in text_features:
-            feature = text_enc_exec_net.infer({'input': feature})['output']
+            feature = text_enc_exec_net.infer({'input': feature})
+            feature = list(feature.values())[0]
```

Comment:
```suggestion
            feature = tuple(feature.values())[0]
```

Reply:
I would suggest `next(iter(feature.values()))`.

Reply:
changed to next(iter(feature.values())).